opentelemetry-prometheus/Cargo.toml

```diff
@@ -21,13 +21,13 @@ rustdoc-args = ["--cfg", "docsrs"]
 
 [dependencies]
 once_cell = { workspace = true }
-opentelemetry = { version = "0.24", default-features = false, features = ["metrics"] }
-opentelemetry_sdk = { version = "0.24", default-features = false, features = ["metrics"] }
+opentelemetry = { default-features = false, features = ["metrics"], path = "../opentelemetry" }
```

Comment:
prometheus was intentionally excluded to use latest sdk to keep 1.0 scope minimized. It is okay to do another release with prometheus picking up newest api/sdk, but please do that into a separate PR, to keep PRs focused on one thing at time.